tools/formatter/configs/yamllint.yml

```diff
@@ -0,0 +1,36 @@
+yaml-files:
+  - '*.yaml'
+  - '*.yml'
+  - '.yamllint'
+
+ignore: |
+  .github/workflows/cla.yml
```

Comment:
Why do we ignore this file?

Reply:
The file `cla.yml` is used in multiple repos organization wide by the admin. If we enforce `yamllint` on this file, we need to reformat it. That makes it different from other `cla.yml`, which makes the maintenance of this file difficult for the admin.
The file `LICENSE.md` is also ignored from `markdownlint` for the same reason.

Reply:
Could you add this info in the file, please?

Reply:
Done